src/script.js

```diff
@@ -1,6 +1,7 @@
-import { storedObjects } from "./utils/localStorageHelper.js";
+import { saveObject, getObject, storedObjects } from "./localStorageHelper.js";
 
 let journalTitleInput, journalDescriptionInput, journalDateInput, journalList;
+let taskTitle, taskDetails, taskDueDate;
```

Comment:
GitHub Actions says that these variables are never used

Reply:
Group 2 added some parameters for tasks, Group 3 can use them to show task list